lib/stm32/common/rtc_common_l1f024.c

```diff
@@ -120,4 +120,14 @@ void rtc_clear_wakeup_flag(void)
 	RTC_ISR &= ~RTC_ISR_WUTF;
 }
 
+/*---------------------------------------------------------------------------*/
+/** @brief Clears the Alarm A flag
+
+@details RTC_ISR[13:8] bits are not write protected: no need to unlock
+*/
+void rtc_clear_alarm_flag(void)
+{
+	RTC_ISR &= ~RTC_ISR_ALRAF;
```

Comment:
As far as I can tell from the F0/l1/f4 and f2 manuals, (and my memory of the "intro to the RTC peripheral" app note that explains some of the platform differences) everythign has alarmA, and some have alarmB, and as far as I can tell,  all the bits for alarm B are always "reserved" on devices that don't have the  alarm.